extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -210,23 +222,25 @@ oppia.directive('graphViz', [
         canAddEdge: '=',
         canDeleteEdge: '=',
         canEditEdgeWeight: '=',
-        canEditOptions: '='
+        canEditOptions: '=',
+        interactionIsActive: '='
```

Comment:
Here and elsewhere, consider `isInteractionActive: '&interactionIsActive` to make it readonly. Then call it using $scope.isInteractionActive()

Reply:
Done